spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -174,6 +174,35 @@ def test_get_calltips(ipyconsole, qtbot):
     control.calltip_widget.hide()
 
 
+@pytest.mark.slow
+@flaky(max_runs=3)
+@pytest.mark.skipif(sys.platform == 'darwin', reason="Times out on macOS")
+def test_get_calltips_documentation_issue_9529(ipyconsole, qtbot):
```

Comment:
Please rename this test `test_get_calltips` and remove the `test_get_calltips` above this one because both are testing almost the same.